certified-connectors/Powell Teams/README.md

```diff
@@ -19,6 +19,8 @@ In order to use this API, you need to be authorized. To do so, click on the "Aut
 - Notes.Read.All
 - Notes.ReadWrite.All
 - Sites.Read.All
+- User.Invite.All
+- User.Read
 - User.Read.All
 

```

Comment:
Please review this [template ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and add the required sections to your Readme.md. All connectors must follow template.